web_development_101/the_front_end/project_html_css.md

```diff
@@ -3246,7 +3246,7 @@ Submit a pull request with a link to your solution in this [file](https://github
 * [rkjenkins2's Solution (Easy)](https://github.com/rkjenkins2/google-homepage) - [View In Browser](https://rkjenkins2.github.io/google-homepage/)
 * [lalfiler's Solution (Easy)](https://github.com/lalfiler/google-homepage) - [View In Browser](https://lalfiler.github.io/google-homepage/)
 </details>
-
+https://github.com/Korak-997/google-website-test
```

Comment:
Notice how your link doesn't look like the other student solutions above it?
You've also put it outside the `</details>` tag.

Here's an example of a properly formatted submission:

`* [lalfiler's Solution (Easy)](https://github.com/lalfiler/google-homepage) - [View In Browser](https://lalfiler.github.io/google-homepage/)`